pkg/dialects/common/enum_adsb_emitter_type.go

```diff
@@ -57,27 +58,38 @@ var labels_ADSB_EMITTER_TYPE = map[ADSB_EMITTER_TYPE]string{
 
 // MarshalText implements the encoding.TextMarshaler interface.
 func (e ADSB_EMITTER_TYPE) MarshalText() ([]byte, error) {
-	if l, ok := labels_ADSB_EMITTER_TYPE[e]; ok {
-		return []byte(l), nil
+	var names []string
```

Comment:
Why was this changed?  This enum is not a bit mask, so these changes to the MarshalText and UnmashalText methods return all the types whose values match the mask, not the single expected string.
For example, I would expect this test to pass:
```			assert.Equal(t, "ADSB_EMITTER_TYPE_ROTOCRAFT", common.ADSB_EMITTER_TYPE_ROTOCRAFT.String())
```
But it fails:
```
-ADSB_EMITTER_TYPE_ROTOCRAFT
+ADSB_EMITTER_TYPE_ROTOCRAFT | ADSB_EMITTER_TYPE_SMALL | ADSB_EMITTER_TYPE_HIGH_VORTEX_LARGE | ADSB_EMITTER_TYPE_HEAVY | ADSB_EMITTER_TYPE_LARGE | ADSB_EMITTER_TYPE_LIGHT | ADSB_EMITTER_TYPE_HIGHLY_MANUV | ADSB_EMITTER_TYPE_NO_INFO
```